packages/examples/src/huge.ts

```diff
@@ -0,0 +1,916 @@
+          },
+          required: ['person']
+        }
+      ]
+    },
+    placeReference: {
+      type: 'object',
+      properties: {
+        original: { type: 'string' },
+        description: { $ref: '#/definitions/uri' }
+      }
+    },
+    coverage: {
+      type: 'object',
+      properties: {
+        spatial: { $ref: '#/definitions/placeReference' },
+        temporal: { $ref: '#/definitions/date' }
+      }
+    },
+    person: {
+      allOf: [
+        { $ref: '#/definitions/subject' },
+        {
+          properties: {
+            private: { type: 'boolean' },
+            gender: { $ref: '#/definitions/gender' },
+            names: {
+              type: 'array',
+              items: { $ref: '#/definitions/name' }
+            },
+            facts: {
+              type: 'array',
+              items: { $ref: '#/definitions/fact' }
+            }
+          }
+        }
+      ]
+    },
+    relationship: {
+      allOf: [
+        { $ref: '#/definitions/subject' },
+        {
+          properties: {
+            type: { $ref: '#/definitions/uri' },
+            person1: { $ref: '#/definitions/resourceReference' },
+            person2: { $ref: '#/definitions/resourceReference' },
+            facts: {
+              type: 'array',
+              items: { $ref: '#/definitions/fact' }
+            }
+          },
+          required: ['person1', 'person2']
+        }
+      ]
+    },
+    sourceDescription: {
+      type: 'object',
+      properties: {
+        id: { type: 'string' },
+        resourceType: { $ref: '#/definitions/uri' },
+        citations: {
+          type: 'array',
+          items: { $ref: '#/definitions/sourceCitation' }
+        },
+        mediaType: { type: 'string' },
+        about: { $ref: '#/definitions/uri' },
+        mediator: { $ref: '#/definitions/resourceReference' },
+        sources: {
+          type: 'array',
+          items: { $ref: '#/definitions/sourceReference' }
+        },
+        analysis: { $ref: '#/definitions/resourceReference' },
+        componentOf: { $ref: '#/definitions/sourceReference' },
+        titles: {
+          type: 'array',
+          items: { $ref: '#/definitions/textValue' }
+        },
+        notes: {
+          type: 'array',
+          items: { $ref: '#/definitions/note' }
+        },
+        attribution: { $ref: '#/definitions/attribution' },
+        rights: {
+          type: 'array',
+          items: { $ref: '#/definitions/resourceReference' }
+        },
+        coverage: { $ref: '#/definitions/coverage' },
+        descriptions: {
+          type: 'array',
+          items: { $ref: '#/definitions/textValue' }
+        },
+        identifiers: {
+          type: 'array',
+          items: { $ref: '#/definitions/identifier' }
+        },
+        created: { type: 'integer' },
+        modified: { type: 'integer' },
+        repository: { $ref: '#/definitions/resourceReference' }
+      },
+      required: ['citations']
+    },
+    agent: {
```

Comment:
This is example shows a couple of validation errors in the example for the agents property, but the error text is empty. Any idea why?